dm/tests/s3_dumpling_lighting/run.sh

```diff
@@ -85,10 +88,9 @@ function run_test() {
 	start_s3
 
 	ps aux | grep dm-master | awk '{print $2}' | xargs kill || true
-	check_master_port_offline 1
+	wait_process_exit dm-master.test
```

Comment:
just found we can use, they'll kill and wait
```
	kill_dm_worker
	kill_dm_master
```

Reply:
ok